package.xml

```diff
@@ -52,51 +52,35 @@
   <!-- Use doc_depend for packages you need only for building documentation: -->
   <!--   <doc_depend>doxygen</doc_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
-
-  <build_depend>actionlib</build_depend>
-  <build_depend>cartesian_control_msgs</build_depend>
-  <build_depend>cartesian_interface</build_depend>
-  <build_depend>controller_manager</build_depend>
-  <build_depend>dynamic_reconfigure</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>hardware_interface</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>trajectory_msgs</build_depend>
-  <build_depend>speed_scaling_interface</build_depend>
-
-  <build_export_depend>actionlib</build_export_depend>
-  <build_export_depend>cartesian_control_msgs</build_export_depend>
-  <build_export_depend>cartesian_interface</build_export_depend>
-  <build_export_depend>controller_manager</build_export_depend>
-  <build_export_depend>dynamic_reconfigure</build_export_depend>
-  <build_export_depend>geometry_msgs</build_export_depend>
-  <build_export_depend>hardware_interface</build_export_depend>
-  <build_export_depend>roscpp</build_export_depend>
-  <build_export_depend>speed_scaling_interface</build_export_depend>
-  <build_export_depend>trajectory_msgs</build_export_depend>
-
-  <exec_depend>actionlib</exec_depend>
-  <exec_depend>cartesian_control_msgs</exec_depend>
-  <exec_depend>cartesian_interface</exec_depend>
-  <exec_depend>controller_manager</exec_depend>
-  <exec_depend>dynamic_reconfigure</exec_depend>
-  <exec_depend>geometry_msgs</exec_depend>
-  <exec_depend>hardware_interface</exec_depend>
-  <exec_depend>roscpp</exec_depend>
-  <exec_depend>speed_scaling_interface</exec_depend>
-  <exec_depend>trajectory_msgs</exec_depend>
+  <depend>actionlib</depend>
+  <depend>cartesian_control_msgs</depend>
+  <depend>cartesian_interface</depend>
+  <depend>cartesian_trajectory_controller</depend>
+  <depend>controller_interface</depend>
+  <depend>controller_manager</depend>
+  <depend>dynamic_reconfigure</depend>
+  <depend>geometry_msgs</depend>
+  <depend>hardware_interface</depend>
+  <depend>pluginlib</depend>
+  <depend>roscpp</depend>
+  <depend>speed_scaling_interface</depend>
+  <depend>trajectory_msgs</depend>
+  <depend>xacro</depend>
```

Comment:
I'm surprised that this is not only a *test_depend*... But ok for me.

Reply:
I do agree.

Reply:
@fmauch I meant the **xacro** package XD. The speed_scaling_interface is in fact a dependency..

Reply:
Yes, I figured ;-)